daemon/cmd/daemon_main.go

```diff
@@ -356,6 +356,12 @@ func initializeFlags() {
 	flags.Bool(option.EnableWireguardUserspaceFallback, false, "Enables the fallback to the wireguard userspace implementation")
 	option.BindEnv(Vp, option.EnableWireguardUserspaceFallback)
 
+	flags.Bool(option.EnableEncryptionStrictMode, false, "Enable encryption strict mode")
+	option.BindEnv(Vp, option.EnableEncryptionStrictMode)
+
+	flags.String(option.EncryptionStrictModeCIDR, "", "Enable encryption strict mode to block all unencrypted traffic to specified CIDR")
```

Comment:
We should be clear that it's not just based on the destination:
```suggestion
	flags.String(option.EncryptionStrictModeCIDR, "", "In strict-mode encryption, all unencrypted traffic coming from this CIDR and going to this same CIDR will be dropped")
```